InventoryBuilder.py

```diff
@@ -183,11 +183,16 @@ def query_vrops(self, vrops):
         return True
 
     def create_resource_objects(self, vrops, token):
+        datacenter_type = "Datacenter"
         for adapter in Resources.get_adapter(target=vrops, token=token):
             if os.environ['DEBUG'] >= '2':
                 print("Collecting vcenter: " + adapter['name'])
             vcenter = Vcenter(target=vrops, token=token, name=adapter['name'], uuid=adapter['uuid'])
-            vcenter.add_datacenter()
+            if datacenter_type == "Datacenter":
+                vcenter.add_datacenter()
+            elif datacenter_type == "CustomDatacenter":
+                vcenter.add_custom_datacenter()
```

Comment:
The `datacenter_type` is not needed. Just run a straight 

```
vcenter.add_datacenter()
vcenter.add_custom_datacenter()
```

If there is a resourcekind `CustomDatacenter` it will return, otherwise it won't.

Reply:
I tried to avoid duplicates in case the VROPS-User has access to normal Datacenters as well as CustomDatacenters. But you are right, it is not needed for the inventory, because the inventory ends up to be a dictionary with the UUID as the key.

One caveat though: In case we get the same cluster from a normal Datacenter and a CustomDatacenter, the `parent_dc_uuid` and `parent_dc_name` will be from the CustomDatacenter.
And parent_dc_name is used as a label in CluserStatsCollector